docs/experts/loading-experts.rst

```diff
@@ -0,0 +1,50 @@
+===============
+Loading Experts
+===============
+
+Some algorithms, such as DAgger, can learn from an expert policy.
+
+In the :doc:`../getting-started/first-steps` tutorial, we first trained an expert and
+then imitate it's behavior using :doc:`../algorithms/bc`.
+In practice, you want to load a pre-trained policy for performance reasons.
```

Comment:
"you want" is a bit strong -- we don't know the user's situation. Could also soften it to "you typicaly want" etc.
```suggestion
In practice, you may want to load a pre-trained policy for performance reasons.
```